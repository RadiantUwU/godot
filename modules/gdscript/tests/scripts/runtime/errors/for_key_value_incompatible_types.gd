func test():
    for u, v: Array in { 1: 1, 2: 2 }:
        prints(u, v)
