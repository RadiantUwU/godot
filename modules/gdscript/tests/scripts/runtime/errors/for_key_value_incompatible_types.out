GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR
>> on function: test()
>> runtime/errors/for_key_value_incompatible_types.gd
>> 2
>> Trying to assign value of type 'int' to a variable of type 'Array'.
